public/app/features/explore/Explore.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import {
+  DataSourceApi,
+  LoadingState,
+  toUtc,
+  DataQueryError,
+  DataQueryRequest,
+  CoreApp,
+  createTheme,
+} from '@grafana/data';
+import { render, screen } from '@testing-library/react';
+import { ExploreId } from 'app/types/explore';
+import { Explore, Props } from './Explore';
+import { scanStopAction } from './state/query';
+import { Provider } from 'react-redux';
+import { configureStore } from 'app/store/configureStore';
+import { AutoSizerProps } from 'react-virtualized-auto-sizer';
+
+const makeEmptyQueryResponse = (loadingState: LoadingState) => {
```

Comment:
we do have an utility to create empty query responses with some default settings you may want to use in https://github.com/grafana/grafana/blob/88ec750728d1991ac69f089a117c359c38d7f802/public/app/features/explore/state/utils.ts#L71